lib/asset_sync/config.rb

```diff
@@ -123,7 +123,15 @@ def manifest_path
     end
 
     def gzip?
```

Comment:
Does the gem still use this after PR merged?
```suggestion
    def gzip?
```

Reply:
Still used in tests. I wanted to avoid breaking changes if it's used downstream somewhere, but happy to remove it if you want.

Reply:
Yes please update those to check `AssetSync.config.compression` instead
I think `gzip?` was tested for internal usage (never mentioned in any doc = I consider as non-public API)